docs/primer/scopeandkeyconcepts.md

```diff
@@ -64,4 +64,10 @@ For more information about links, see the Link reference.
 
 ## Geospatial data
  
- To follow
\ No newline at end of file
+Geospatial data contains information about objects and their location on Earth. A 'position' in geospatial data describes a single location. Each position contains information about longitude, latitude, and (sometimes) altitude or elevation.
+
+Geospatial data is one of the main elements of open fibre data. When location information is available, open fibre data represents nodes as a single position, and links as two or more positions.
```

Comment:
A slight copy edit based on https://www.ibm.com/topics/geospatial-data. Rather than introducing positions, I think it would be better to introduce the concept of a geometry and the two geometry types used in the standard. We could then give brief examples the geometry used to represent a nodes and links, e.g. for a node:

```json
{
  "geometry": {
    "type": "Point",
    "coordinates": [
      26.081,
      -24.405
    ]
  }
}
```

We can display those using [sphinx-panels](https://sphinx-panels.readthedocs.io/en/latest/#tabbed-content) so that they don't take up too much room. Could you have a go at making those changes?

To begin with, we can just embed the examples as a code block in each panel, but, if we decide to pursue https://github.com/Open-Telecoms-Data/open-fibre-data-standard/issues/57, then they can be pulled in from the schema so that they don't go out of date.

```suggestion
Geospatial data is information that describes objects or features with a location on or near the surface of the earth. Geospatial data typically combines location information (usually coordinates on the earth) and attribute information (the characteristics of the object concerned). 

OFDS data is geospatial data. It can contain both location information, such as the location of nodes and links, and attribute information, such as the capacity of a link.
```